core/templates/pages/exploration-player-page/layout-directives/progress-nav.directive.html

```diff
@@ -48,10 +48,6 @@
 
   progress-nav .progress-nav-bar {
     background-color: white;
-    margin-left: 87px;
-    /*
-    87px is used to move the opppia avatar inside the conversation-skin-tutor-card div (so that it does not goes out of screen during zoom in).
```

Comment:
Comments generally go above the line it talks about. Also, typo "opppia" -> "Oppia" and make the line follow 80 char max limit. Thanks!

Reply:
Hey @kevintab95 I think you misunderstood, this part of the code is being deleted and not added.

Reply:
Yikes! Thanks for pointing it out.